src/prompts/index.tsx

```diff
@@ -41,34 +36,31 @@ export type PromptManagerState =
 // postaction -> ready
 
 export class PromptManager {
-    private readonly kumulosClient: Kumulos;
     private readonly context: Context;
     private readonly pushContainer: RootFrameContainer;
     private readonly triggerFilter: PromptTriggerEventFilter<PushPromptConfig>;
 
     private state?: PromptManagerState;
     private subscriptionState?: PushSubscriptionState;
-    private prompts: PromptConfigs<PushPromptConfig>;
     private activePrompts: PushPromptConfig[];
     private currentlyRequestingPrompt?: PushPromptConfig;
     private pushOpsManager?: PushOpsManager;
-    private channels: Channel[];
     private ui?: Ui;
-    private platformConfig?: PlatformConfig;
+    private publicKey?: string;
+    private prompts: PromptConfigs<PushPromptConfig>;
     private currentPostAction?: PromptAction;
     private channelSubscriptionManager?: ChannelSubscriptionManager;
 
-    constructor(client: Kumulos, ctx: Context, rootFrame: RootFrame) {
-        this.prompts = {};
+    constructor(ctx: Context, rootFrame: RootFrame, publicKey?: string, prompts?: PromptConfigs<PushPromptConfig>) {
```

Comment:
If we remove loadPrompts we can remove publicKey as well

Reply:
Removed